src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -872,6 +875,15 @@ public sealed partial class VariantWrapper
         public VariantWrapper(object obj) { }
         public object WrappedObject { get { return default(object); } }
     }
+    public static partial class RuntimeEnvironment
+    {
+        public static string SystemConfigurationFile {get { return default(string); } }
+        public static bool FromGlobalAccessCache(System.Reflection.Assembly a) { return default(bool); }
+        public static string GetRuntimeDirectory() { return default(string); }
+        public static System.IntPtr GetRuntimeInterfaceAsIntPtr(System.Guid clsid, System.Guid riid) { return default(System.IntPtr); }
+        public static object GetRuntimeInterfaceAsObject(System.Guid clsid, System.Guid riid) { return default(object); }
+        public static string GetSystemVersion() { throw null; }
```

Comment:
Nit: Should this be `return default(string);` instead of `throw null;` to be consistent with the other members?